deploy/999_impersonate_owners.ts

```diff
@@ -30,6 +31,11 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
     ALETH: ["0x1d2c4cd9bee9dfe088430b95d274e765151c32db"],
     WETH: ["0xceff51756c56ceffca006cd410b03ffc46dd3a58"],
     SETH: ["0xc5424b857f758e906013f3555dad202e4bdb4567"],
+    // D4
+    ALUSD: ["0x43b4fdfd4ff969587185cdb6f0bd875c5fc83f8c"],
+    FEI: ["0x94b0a3d511b6ecdb17ebf877278ab030acb0a878"],
```

Comment:
this appears to be the FEI/ETH uniswap pool https://etherscan.io/address/0x94b0a3d511b6ecdb17ebf877278ab030acb0a878, correct address is `0x956F47F50A910163D8BF957Cf5846D573E7f87CA`